### DISCUSSION TITLE
Support lifecycle of ldap2pg

### DISCUSSION THREAD

**Discussion by kato-sho** [dalibo/ldap2pg#618] [General]
I'm looking for a support policy of ldap2pg in the ldap2pg documents, but I don't find this infomation.
I would appreciate it if you could answer the following questions.

- How long is the support lifecycle of ldap2pg?
- Does ldap2pg v6.0 support PostgreSQL 16?

**Comment by bersace** [comment]
Hi @kato-sho . sorry for the late reply.

ldap2pg 6.1 will support Postgres 16 and will land this spring.

> How long is the support lifecycle of ldap2pg?

Not sure about the sens of this question.
We don't maintain ldap2pg 5.
The base policy is to ship a new version at least every year after Postgres major release, tested against it.
